charts/jaeger/values.yaml

```diff
@@ -466,6 +466,7 @@ collector:
     # -- ServiceMonitor metric relabel configs to apply to samples before ingestion
     # https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/api.md#endpoint
     metricRelabelings: []
+  initContainers: []
```

Comment:
Wouldn't adding this property here mean that the `{{- if .Values.collector.initContainers }}` condition always evaluates to true? I guess we should comment out this line, right?

Reply:
@pavelnikolov can you review it again?